plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/sns.ts

```diff
@@ -41,7 +41,7 @@ export class SnsServiceExtension implements ServiceExtension {
         MessagingDestinationKindValues.TOPIC;
       const { TopicArn, TargetArn, PhoneNumber } = request.commandInput;
       spanAttributes[SemanticAttributes.MESSAGING_DESTINATION] =
```

Comment:
@alingamn  should the key be messaging.destination.name per latest spec?

Reply:
@pavankrish123 Semantic attribute messaging.destination.name is not yet available.  This is the PR to address that https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1727

Reply:
Thanks Anusha